docs/architecture/deployment/installation-guide.md

```diff
@@ -149,3 +149,55 @@ helm -n <namespace> install prophecy prophecy/prophecy-installer --version <prop
 --set global.prophecy.wildcardCert.name=<wildcard cert secret name> --set athena.controlcenter.disabled=true
 --set global.repository=<Image repository> --set global.prophecy.imagePullSecret=<Image pull secret name> --set athena.isDarkCluster=true
 ```
+
+#### When an external SQL database is provided
+
+1. Create a external SQL Database (e.g. Google SQL instance).
+2. Run the following commands by connecting to the external SQL Database.
+
+```
+CREATE USER sdl WITH PASSWORD '<custom-sdl-password>';
+
+CREATE DATABASE gogs;
+GRANT ALL PRIVILEGES ON DATABASE gogs TO sdl;
+
+CREATE DATABASE federator;
+GRANT ALL PRIVILEGES ON DATABASE federator TO sdl;
+```
+
+3. Create a Kubernetes secret with the following data.
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <env-secrets-file-name>
+  namespace: <namespace>
+type: Opaque
+stringData:
+  envfile: |-
+    ES_USERNAME="{elastic-search-user}"
```

Comment:
Little concern here , we are sharing all our password env. names in public docs .. Need to think a bit